src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/entities/BailCaseFieldDefinition.java

```diff
@@ -8,8 +8,22 @@
 import uk.gov.hmcts.reform.bailcaseapi.domain.entities.ccd.field.YesOrNo;
 
 public enum BailCaseFieldDefinition {
+    APPLICATION_SENT_BY(
+        "sentByChecklist", new TypeReference<String>() {}),
     IS_ADMIN(
         "isAdmin", new TypeReference<String>() {}),
+    APPLICANT_GIVEN_NAMES(
+        "applicantGivenNames", new TypeReference<String>() {}),
+    APPLICANT_FAMILY_NAME(
+        "applicantFamilyName", new TypeReference<String>() {}),
+    APPLICANT_DOB(
+        "applicantDateOfBirth", new TypeReference<String>() {}),
+    APPLICANT_GENDER(
+        "applicantGender", new TypeReference<String>() {}),
+    APPLICANT_GENDER_OTHER(
+        "applicantGenderEnterDetails", new TypeReference<String>() {}),
+    APPLICANT_NATIONALITIES(
```

Comment:
There is one more field to capture whether the applicant has a nationality or is stateless. Can you please check and add that here.

Reply:
Added the missing APPLICANT_NATIONALITY field